File: src/main/java/uk/co/threebugs/mochiwhattotrade3/setupgroup/SetupGroupDto.java
package uk.co.threebugs.mochiwhattotrade3.setupgroup;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;
import uk.co.threebugs.mochiwhattotrade3.setupgroups.SetupGroupsDto;

import java.nio.file.Path;


@Builder
@Value
@Jacksonized
public class SetupGroupDto {

    Integer id;
    SetupGroupsDto setupGroups;
    Path path;
    String symbol;
    Boolean enabled;
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/setupgroup/SetupGroupDto.kt
package uk.co.threebugs.mochiwhattotrade3.setupgroup

import uk.co.threebugs.mochiwhattotrade3.setupgroups.SetupGroupsDto
import java.nio.file.Path


data class SetupGroupDto(
    var id: Int? = null,
    var setupGroups: SetupGroupsDto? = null,
    var path: Path? = null,
    var symbol: String? = null,
    var enabled: Boolean? = null)

